src/main/java/com/olgaboiar/mint/Server.java

```diff
@@ -26,14 +26,14 @@ public Server(String host, int port) {
     @Override
     public void start() throws IOException {
         serverSocket = new ServerSocket(port);
+        logger.logToFile("\nConnection on port " + port, "logs.txt");
```

Comment:
Can we assert this in unit test?

Reply:
addressed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/ed436b97c97624e8c2bd6ed9948bcd67ec5967f0